BreastCancer/TasksAndSteps/APHExerciseCheckinTask.m

```diff
@@ -0,0 +1,560 @@
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+#import "APHExerciseCheckinTask.h"
+
+//  Exercise Motivator Task Items
+static NSString* const kMotivatorTaskID                 = @"4-APHExerciseSurvey-7259AC18-D711-47A6-ADBD-6CFCECDED1DF";
+static NSString* const kWalkFiveThousandSteps           = @"Walk 5,000 steps every day";
+static NSString* const kExerciseEverySingleDay          = @"Exercise Every Single Day for at least 30 minutes";
+static NSString* const kExerciseThreeTimesPerWeek       = @"Exercise at least 3 times per week for at least 30 minutes";
+static NSString* const kWalkTenThousandSteps            = @"Walk 10,000 steps at least 3 times per week";
+static NSString* const kExerciseEverySingleDayImage     = @"banner_exersiseeveryday";
+static NSString* const kExerciseThreeTimesPerWeekImage  = @"banner_exersise3x";
+static NSString* const kWalkFiveThousandStepsImage      = @"banner_5ksteps";
+static NSString* const kWalkTenThousandStepsImage       = @"banner_10ksteps";
+static NSString* const kImageResultKey                  = @"exercisesurvey101";
+
+// constants for result queries
+static NSString* const kAPCTaskAttributeUpdatedAt   = @"updatedAt";
+
+//  task identifier
+static NSString* const kExerciseCheckInID   = @"Weekly Exercise Check-in";
+
+//  step index
+static NSInteger const kNegativeStepIndex      = -1;
+static NSString* const kInstructionStepIndex   = @"0";
+static NSString* const kUserProgressStepIndex  = @"1";
+static NSString* const kUserSucceededStepIndex = @"2";
+static NSString* const kUserInputStepIndex     = @"4";
+static NSString* const kUserFailedStepIndex    = @"3";
+static NSString* const kUserUncertainIndex     = @"5";
+static NSString* const kReasonsStepIndex       = @"6";
+static NSString* const kReasonsTwoStepIndex    = @"7";
+
+//  step identifiers
+static NSString* const kInstructionStepID   = @"instruction";
+static NSString* const kUserProgressStepID  = @"progress";
+static NSString* const kUserSucceededStepID = @"success";
+static NSString* const kUserFailedStepID    = @"failure";
+static NSString* const kUserInputStepID     = @"failure.reason";
+static NSString* const kUserUncertainID     = @"uncertain";
+static NSString* const kReasonsStepID       = @"reasons";
+static NSString* const kReasonsTwoStepID    = @"reasons.two";
+
+//  localized text
+static NSString* const kInstructionTitle    = @"Exercise Motivator Check-in";
```

Comment:
Odd that all this text, given it'll need localizing, it not at either in a separate strings file or loaded from a localizable resource.